### PR TITLE
feat(XTD-348): Stripe success param

### PR DESCRIPTION
**Pull request by tristandubbeld**
Shows notification when a subscription through Stripe has been successful. Based on if the subscription webhook from Stripe came through fast enough we well show the user more information and an action.

### Testing

1. Open the dashboard with a team without trial
2. Start a trial
3. When redirected by Stripe, it should show a notification on the dashboard

Based on if your internet speed and device are faster than the webhook, you should see different notifications. When the webhook didn't come yet in you should see a refresh action in the notifications. To mock the refresh action notification, you can also open the dashboard for a team without trial and add `&stripe=success` to the url. To mock the notification without action, you can add `&stripe=success` to the url for a team that already has a subscription.

### Screenshots

**When webhook didn't come through yet**
image

**When webhook came through and subscription is available**
image

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/7248/commits/45a58d5cef3bd65a3962bfae59a4cab950ef1862 is at https://pr7248.build.csb.dev/s/new.

File: packages/app/src/app/pages/Dashboard/index.tsx [hunk 1]
```diff
@@ -53,12 +54,43 @@ export const Dashboard: FunctionComponent = () => {
...
+        title: 'Successfully activated subscription',
+        message: isProDelayed
+          ? 'Please reload to update the dashboard.'
+          : undefined,
+        actions: isProDelayed
+          ? {
+              primary: {
+                label: 'Reload',
+                run: () => {
+                  actions.getActiveTeamInfo();
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Dashboard/index.tsx:85]: Refetches the team info which updates the UI when subscription is truthy.

**Review by olarclara — COMMENTED** [review]
Is this testable yet? I'm unable to see a notification:

Screen Shot 2022-12-19 at 17 38 35

**Review by olarclara — APPROVED** [review]
LGTM!

**Comment by tristandubbeld** [comment]
> I didn't notice that I was codesandbox.stream instead of the actual preview.

Oh yeah I should have actually mentioned that, sorry @olarclara! Stripe always redirects to stream for some reason. We can test this on stream if we want, but also double check it on production?

**Comment by tristandubbeld** [comment]
Deployed this to stream and verified that it works :)

**Pull request merged**